packages/accounts-password/password_server.js

```diff
@@ -538,44 +538,89 @@ Accounts.sendResetPasswordEmail = function (userId, email) {
   if (!email || !_.contains(_.pluck(user.emails || [], 'address'), email))
     throw new Error("No such email for user.");
 
+  var tokenReason;
```

Comment:
I am not used to brace-less ifs, so I missed that.

But I was just answering why `var`. I do not really care.